hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/KeyValueContainerUtil.java

```diff
@@ -169,7 +169,7 @@ public static void parseKVContainerData(KeyValueContainerData kvContainerData,
     File metadataPath = new File(kvContainerData.getMetadataPath());
 
     // Verify Checksum
-    ContainerUtils.verifyChecksum(kvContainerData);
+    ContainerUtils.verifyChecksum(kvContainerData, config);
```

Comment:
Any reason why moved the config check into the method? If we keep it here, it is more readable.

`if (enabled) {
verifyChecksum
}`

What do you think?

Reply:
I thought the checksum verification enable/ disable can be inside the verifychecksum function, so that every time we invoke the function we don't have to enable/disable outside the function call. What are your thoughts on that?